File: 卡码网/Python/test02.py
#2. A+B问题II
import sys

def main():
	for line in sys.stdin:
		n=int(line)
		for i in range(n):
			[a,b]=map(int,sys.stdin.readline().split())
			print(a+b)

if __name__=="__main__":
	main()

File: 卡码网/Js/test02.js
//2. A+B问题II
const rl=require("readline").createInterface({input:process.stdin});
const iter=rl[Symbol.asyncIterator]();
const readline=async ()=>(await iter.next()).value;

async function main(){
  while(line=await readline()){
    let n=parseInt(line);
    for(let i=0;i<n;i++){
      const [a,b]=(await readline()).split(" ").map(Number);
      console.log(a+b);
    }
  }
}

main();